ipatests/test_integration/test_installation.py

```diff
@@ -1346,6 +1346,37 @@ def test_KRA_install_after_cert_renew(self):
         self.master.run_command(['systemctl', 'start', 'chronyd'])
 
 
+class TestKRAinstallOnReplicaWithCAHost(IntegrationTest):
+    """ Test that KRA install on replica with ca_host overriden fails
+
+    KRA install on a replica should fail
+    if the ca_host line in /etc/ipa/default.conf is present
+
+    Related: https://pagure.io/freeipa/issue/8245
+    """
+
+    num_replicas = 1
+
+    def test_kra_install_on_replica_with_ca_host_overriden(self):
+        tasks.install_master(self.master)
+        tasks.install_replica(self.master, self.replicas[0])
+
+        content = self.replicas[0].get_file_contents(paths.IPA_DEFAULT_CONF,
+                                                     encoding='utf-8')
+        ca_host_line = "ca_host = %s" % self.master.hostname
+        new_content = content + '\n' + ca_host_line
+        self.replicas[0].put_file_contents(paths.IPA_DEFAULT_CONF,
+                                           new_content)
+
+        self.master.run_command(['firewall-cmd', '--add-port=8443/tcp'])
```

Comment:
I'm not sure there, we seem to be fine with installing KRAs without using this.

Reply:
Actually, my mistake there. This line is needed when ca_host is overriden so that the replica can contact the first server. Could you please re-add it?